### PR TITLE
Pre release fixes

### LINKED ISSUE #382: Clean UI inconsistencies
Here's the list of UI issues left to fix (thanks @LeeAlephium !):

- [x] Don't display outlines on click, but continue to display them when selected with tab (accessibility) (https://github.com/alephium/desktop-wallet/issues/378#issuecomment-1261351090)
- [x] Increase spacing between input fields (https://github.com/alephium/desktop-wallet/issues/378#issuecomment-1261400865)
- [x] Change recovery phrase message to "Make sure to *store* the words in a secure location..." (https://github.com/alephium/desktop-wallet/issues/378#issuecomment-1261430921) French translation is already correct.
- [x] Increase line height a bit on paragraphe (like the one mentioned here https://github.com/alephium/desktop-wallet/issues/378#issuecomment-1261430921)

### PR DESCRIPTION
**Pull request by nop33**
Closes #382

### CODE REVIEW

**Comment by LeeAlephium** [comment]
It doesn't make sense to show the "store your mnemonic" message when asking to import (which I wrote here https://github.com/alephium/desktop-wallet/issues/378#issuecomment-1261430921)

File: src/components/ToggleSection.tsx [hunk 1]
```diff
@@ -39,20 +39,20 @@ const ToggleSection = ({ title, onClick = () => null, children, className }: Tog
...
           opacity: isShown ? 1 : 0,
           visibility: isShown ? 'visible' : 'hidden'
         }}
         transition={{ duration: 0.2 }}
       >
-        <CellChildrenInner>{children}</CellChildrenInner>
-      </CellChildren>
+        <Children>{children}</Children>
+      </Content>
     </div>
```
Comment (LeeAlephium) [inline src/components/ToggleSection.tsx:56]: Unnecessary renames

**Comment by LeeAlephium** [comment]
> It's a reminder. We want to remind users that their phrase is really really important and should be stored in a secure location.

How can it be a reminder when it must already be stored in order to import anything

File: src/components/ToggleSection.tsx (hunk 1 above)
Comment (mvaivre) [reply src/components/ToggleSection.tsx:56]: Why so cold ? :/ 

The component names are shorter, so why not.

NIT: If it was me I'd just rename "Content" into "SectionContent" for extra clarity, as a mention to the ToogleSection name.

**Comment by mvaivre** [comment]
> How can it be a reminder when it must already be stored in order to import anything

I don't know, if they suddenly realise that having the words sticked to the fridge or on a post-it on a wall might indeed no be a good idea.

**Review by LeeAlephium — APPROVED** [review]

**Pull request merged**